hardware/neopixel/neopixel.js

```diff
@@ -45,6 +47,9 @@ module.exports = function(RED) {
         var p4 = /^[0-9]+,[0-9]+,[0-9]+,[0-9]+,[0-9]+$/
 
         function inputlistener(msg) {
+            if (msg.hasOwnProperty("brightness")) {
+                node.child.stdin.write("brightness,"+msg.brightness.toString()+"\n"); 
```

Comment:
might want to check it really is 0 - 100 here...

Reply:
I thought the earlier checks would make sure it was an number at this point but I'll re-look at it.  The python code does the full bounds checking/limiting at the moment

Reply:
maybe it does - but stops anyone doing something dumb and passing some large object over by mistake.

Reply:
Done the bounds checking at same time as bounds checking done for wipe time